api/src/main/java/gov/cms/ab2d/api/service/JobServiceImpl.java

```diff
@@ -36,4 +36,17 @@ public Job createJob(String resourceTypes, String url) {
 
```

Comment:
Also into `domain`

Reply:
In that case we would have a design problem. JobService won't need to access any API- or Worker-specific services.